Packs/Logzio/Integrations/Logzio/README.md

```diff
@@ -0,0 +1,1725 @@
+
+
+## Additional Information
```

Comment:
either fill in or remove empty sections
also please remove the possible errors section